main.go

```diff
@@ -139,7 +139,7 @@ func mainWithError() error {
 	daemonCommand := newCommand.DaemonCommand().CobraCommand()
 
 	daemonCommand.PersistentFlags().String(f.Service.Configurer.ConfigMap.Key, "values", "Key in configmap holding values data.")
-	daemonCommand.PersistentFlags().String(f.Service.Configurer.ConfigMap.Name, "draughtsman-values", "Name of configmap holding values data.")
+	daemonCommand.PersistentFlags().String(f.Service.Configurer.ConfigMap.Name, "draughtsman-values-configmap", "Name of configmap holding values data.")
```

Comment:
This is the right name. See https://github.com/giantswarm/opsctl/blob/04fd25a56ba4c95f4e09d0ea9ba35388f5b2646e/command/create/draughtsman/command.go#L29.